README.md

```diff
@@ -115,11 +115,17 @@ provider.
 Set required environment variables based as per your infrastructure settings
 
 ```sh
+$ # API token provided from CSP
 $ export API_TOKEN=xxx
+$ # Id of a VMC Org in which test SDDC are (to be) placed 
 $ export ORG_ID=xxxx
+$ # Id of an existing SDDC used for SDDC data source (import) test
 $ export TEST_SDDC_ID=xxx
+$ # Name of above SDDC
```

Comment:
Isn't the ID (TEST_SDDC_ID) sufficient?

Reply:
It is sufficient to import an SDDC, however the name is used in an attribute check post-import, since this is the only attribute check I've inherited I'd prefer to leave it as is. In the future more attribute checks will make the import test even more useful.